File: Selenium/Project/Activity5.java
package project;

import static org.testng.Assert.assertEquals;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity5 {
	//Declare webdriver
	WebDriver driver;
	
	//Set up method
	@BeforeClass
	  public void setUp() {
		   {
		        // Initialize webdriver
		        driver = new FirefoxDriver();
		        //Open the page
		        driver.get("https://alchemy.hguy.co/jobs");
		  }
	  }
	
	
  @Test(priority = 1)
  public void jobsPageTest() {
	  driver.findElement(By.id("menu-item-24")).click();
	  String jobsheading = driver.findElement(By.id("menu-item-24")).getText();
	  System.out.println("Page Title " + jobsheading);
	  System.out.println("Page Title " + driver.getTitle());
      assertEquals(driver.getTitle(), "Jobs – Alchemy Jobs");
	  
  }
  
  
  @AfterClass
  public void tearDown() {
      // Close the browser
      driver.quit();
  }
  
}

File: Selenium/Project/Activity5.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
with webdriver.Firefox() as driver:
    # Navigate to the URL
    driver.get("https://alchemy.hguy.co/jobs")

    # Navigate to Jobs page
        
    driver.find_element(By.ID, "menu-item-24").click()
    page_heading = driver.find_element(By.ID, "menu-item-24")
    print("Page heading is: ", page_heading.text)
    
    # Print the title of the page
    print("Page title is: ", driver.title)